### DISCUSSION TITLE
Preliminary Specimens in EMu

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#283] [HOW TO...]
Exhibitions ALWAYS needs a catalogue record to attach to events. Records for specimens, objects and cultural resources should be created by the relevant collections/registration team for exhibitions to use. In Natural History, if this is not possible for some reason Exhibitions may create a preliminary record.
 
* googledoc workflow [How to setup and transfer preliminary specimens](https://docs.google.com/document/d/1a5wGVmdpZuJlMhY7RHtEivG-ETPltkjV-jDh1Yg3EFs/edit#heading=h.hcyfxuh7sqbd)

**Comment by fmjjones** [comment]
Workflow for creating child records from an existing parent record: https://docs.google.com/document/d/1xGIcz9iafC2q1eB6Lqbvq5c1coVn9QSm21oi1KhLlN8/edit?usp=sharing
